cdap-ldap-role/src/main/java/io/cdap/cdap/security/authorization/ldap/role/permission/RolePermissionConverter.java

```diff
@@ -102,12 +102,17 @@ public static List<EntityTypeWithPermission> convertToEntityTypeWithPermission(R
           new EntityTypeWithPermission(EntityType.APPLICATION, StandardPermission.CREATE),
           new EntityTypeWithPermission(EntityType.DATASET, StandardPermission.CREATE),
           new EntityTypeWithPermission(EntityType.APPLICATION, StandardPermission.GET),
-          new EntityTypeWithPermission(EntityType.ARTIFACT, StandardPermission.GET)
+          new EntityTypeWithPermission(EntityType.ARTIFACT, StandardPermission.GET),
+          new EntityTypeWithPermission(EntityType.SYSTEM_APP_ENTITY, StandardPermission.CREATE),
```

Comment:
Does creating pipeline need to have create/update permissions on system app entities?

Reply:
removed